buildSrc/src/test/java/org/springframework/boot/build/context/properties/ConfigurationPropertiesTests.java

```diff
@@ -32,7 +32,7 @@
 class ConfigurationPropertiesTests {
 
 	@Test
-	void whenJsonHasAnIntegerDefaultValueThenItRemainsAnIntegerWhenRead() {
+	void whenJsonHasAnIntegerDefaultValueThenItRemainsAsIntegerWhenRead() {
```

Comment:
`An` is correct here, IMO.

Reply:
@wilkinsona Thanks for the quick feedback!

I didn't know that a noun could come after "remain". After searching it a bit, I found I was wrong. I reverted it in 2420f0c.